jaxrs-tck/pom.xml

```diff
@@ -236,5 +103,71 @@
             <version>1.4</version>
         </dependency>
 
+        <!-- Test Dependencies -->
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <scope>test</scope>
+        </dependency>
+
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-compiler-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <artifactId>maven-surefire-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
It is a best practice to not add that particular group ID as it is the default group ID of Maven.

Reply:
I guess that's debatable. The default value in the XSD is for `groupId` is `org.apache.maven.plugins`. I honestly don't mind either way, so if we prefer to have it I can make sure to add it everywhere. If we prefer to not have it defined, I can make sure they are not used anywhere.

Reply:
My personal vote would be to go with the Maven best practice of CoC and *neven* copy a default value explicitly.

Reply:
I went ahead and removed all the `<groupId>org.apache.maven.plugins</groupId>`.

Reply:
This conversation appears to be resolved so marking it as such.